app/controllers/api/v1/orders_controller.rb

```diff
@@ -96,8 +96,23 @@ def destroy
         render json: {}
       end
 
+      def summary
+        active_orders = Order.orders_group_by_state(is_priority: false)
+        active_orders_count = orders_count(active_orders)
+
+        priority_orders = Order.orders_group_by_state(is_priority: true).transform_keys{ |key| "priority_".concat(key) }
```

Comment:
@abulsayyad123 it will be good to keep these 2 methods different. It's difficult to understand. 

e.g first method `active_orders_count` and other is `priority_active_orders_count ` . Then summary method will be like: 

`active_orders_count.merge!(priority_active_orders_count)
        render json: active_orders_count`